shortcuts.js

```diff
@@ -0,0 +1,391 @@
+const AppConstants = require("resource://gre/modules/AppConstants.jsm");
```

Comment:
This file looks like copy and paste from a gecko file. Can you please commit it exactly has it is in gecko in a separate commit and then add the changes in a separate commit? This will make the view easier.